raiden/transfer/token_network.py

```diff
@@ -254,30 +255,19 @@ def handle_batch_unlock(
     block_number: BlockNumber,
     block_hash: BlockHash,
 ) -> TransitionResult:
-    events = list()
```

Comment:
thats a lot of business logic removed

could you explain why? 

note that, really important stuff like the removal of a channel is gone

`             del token_network_state.channelidentifiers_to_channels[state_change.participant][channel_state.identifier] `

Reply:
subdispatch_to_channels_by_participant_address calls subdispatch_to_channels, which already does that if the channel must be deleted.

Reply:
What I mean is, the deleted code was duplicated